sdcm/tester.py

```diff
@@ -1355,29 +1355,32 @@ def get_cluster_docker(self):
 
     def get_cluster_baremetal(self):
         # pylint: disable=too-many-locals,too-many-statements,too-many-branches
-
+        baremetal_info: cluster_baremetal.BareMetalCredentials = KeyStore().get_baremetal_credentials()
```

Comment:
you gonna have multiple of those, and not just one set of configuration... 
probably need to name them and have a configuration option to point to them

similar to how we do it fro the internal-qa repo, as in the 5000 table case:

```yaml
cs_user_profiles:
    - scylla-qa-internal/cust_d/templated_tables_mv.yaml
```

Reply:
Yes, I thought of that. I can implement this in this PR if needed.

Reply:
Done, implemented as `s3_baremetal_config`